microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -59,15 +59,13 @@ Mobile devices have much less CPU power than desktops and laptops.  Whenever you
 
 1. In DevTools, open the **Performance** tool.
 
```

Comment:
```suggestion
1. Make sure the **Screenshots** checkbox is selected.

```
is the checkbox required to be selected?  if so, restore step, w/ this wording.

Reply:
Yes, simulating a mobile CPU, has nothing to do with Screenshots. It doesn't matter whether it is enabled or not. Hence, Step 2 is not required, and hence, removed it from the doc.

Reply:
step isn't needed